storage/src/tests/distributor/statecheckerstest.cpp

```diff
@@ -128,17 +134,17 @@ struct StateCheckersTest : public CppUnit::TestFixture,
         std::ostringstream ost;
 
         c.siblingBucket = getIdealStateManager().getDistributorComponent()
-                .getSibling(c.bucketId);
+                          .getSibling(c.bucket.getBucketId());
```

Comment:
The expression `c.bucket.getBucketId()` is used a lot, could add a forwarding `getBucketId()` to `StateChecker::Context` itself to avoid a visible level of indirection for these.